ait/core/server/handler.py

```diff
@@ -84,3 +85,51 @@ def handle(self, input_data):
             tuple of packet UID and message received by stream
         """
         return pickle.dumps((self._pkt_defn.uid, input_data), 2)
+
+
+class CCSDSPacketHandler(Handler):
+
+    def __init__(self, input_type=None, output_type=None, **kwargs):
+        """
+        Params:
+            input_type:   (optional) Specifies expected input type, used to
+                                     validate handler workflow. Defaults to None.
+            output_type:  (optional) Specifies expected output type, used to
+                                     validate handler workflow. Defaults to None
+            **kwargs:     (required) APID values: packet name pairs
```

Comment:
Instead of specifying `**kwargs` here it would be good to specify what keyword arguments are required / optional.

For example,
```
packet_types: (required) APID values : packet name pairs
packet_secondary_header_length: (optional) ...
```

Otherwise someone reading the docs has to go look in the code to figure out what the name for the keyword arguments you're describing are.